fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -416,10 +417,12 @@ protected void buildFromClause(StringBuilder fromClause, String simpleName) {
         fromClause.append(" LR ");
 
         // This is requires for the count queries here
-        fromClause.append(JOIN);
-        processFromClauseForLastUpdated(fromClause, simpleName);
-        fromClause.append(" R ON R.RESOURCE_ID = LR.CURRENT_RESOURCE_ID ")
-                .append(" AND R.IS_DELETED = 'N' ");
+        if (!queryParmLastUpdateds.isEmpty()) {
+            fromClause.append(JOIN);
+            processFromClauseForLastUpdated(fromClause, simpleName);
+            fromClause.append(" R ON R.RESOURCE_ID = LR.CURRENT_RESOURCE_ID ")
+                    .append(" AND R.IS_DELETED = 'N' ");
+        }
```

Comment:
this is basically the only change I was trying to make...avoid the join with the xx_RESOURCES tables (unless we have a lastUpdated query because that was a bigger change than I wanted to take on)